app/assets/stylesheets/user.css.scss

```diff
@@ -0,0 +1,25 @@
+.user-background {
+  padding-top: 1%;
+  padding-bottom: 1%;
+  background-color: black;
+}
+
+.user-header {
+  text-align: center;
+  background-color: #170563;
```

Comment:
Color literals like `#170563` should only be used in variable declarations; they should be referred to via variable everywhere else.